front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputsTable/Export.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { render, screen, fireEvent } from 'utils/testUtils/rtl';
+import { requestBlob } from 'utils/request';
+
+import views from 'modules/commercial/insights/fixtures/views';
+import Export from './Export';
+
+const viewId = '1';
+const apiPath = '/web_api/v1/insights/views/1/inputs/as_xlsx';
+const application =
+  'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet';
+const mockViewData = views[0];
+let mockLocationData: { query: Record<string, unknown> } = { query: {} };
+
+jest.mock('modules/commercial/insights/hooks/useInsightsView', () => {
+  return jest.fn(() => mockViewData);
+});
+
+jest.mock('utils/cl-intl');
+jest.mock('hooks/useLocale');
+jest.mock('services/locale');
```

Comment:
Curious why do you mock both the hook and service?

Reply:
The service also got imported and needed to be mocked, I think it was from the `helperUtils` file.